### PR TITLE
refactor: Resolve name collision of route target attribute

### LINKED ISSUE #14: Rename "target" attribute of Link component to "to"
The current naming of attribute for target route leads to name collision with [Anchor element's attribute ](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a#target) attribute, leading to both confusion and impossibility to expose that attribute to library user.

It would also help people transitioning from / to yew's router to choose the same name - `to`

This is a breaking change, so if new breaking release isn't planned, there could be a version released that takes both attributes, but that leads to issue of which one should take priority.

### PR DESCRIPTION
**Pull request by kate-shine**
resolves #14

I hope I didn't miss something :)

Not sure what to do with `Active` component. `to` sounds wrong in that context, leaving it as `target` is probably fine despite inconsistency, both have a different purpose. Also, `span` doesn't have a `target` attribute, so no html collision in this case.

### CODE REVIEW

**Comment by ctron** [comment]
I guess the examples, and the `Active` component.

**Comment by kate-shine** [comment]
Ugh, I'm sorry about that. As I stated in the first post, I'm not so sure about `Active` :/ `to` doesn't seem like it makes sense semantically, `target` sounds fine.

**Comment by kate-shine** [comment]
Another option is to rename that attribute for both to `route`, which solves the ambiguity issue, but makes it syntactically different from yew_router (dunno if that feature has any value for you though)

**Comment by ctron** [comment]
> Ugh, I'm sorry about that. As I stated in the first post, I'm not so sure about `Active` :/ `to` doesn't seem like it makes sense semantically, `target` sounds fine.

Naming is hard … You're right. It doesn't. Keeping it `target` also feels wrong. A good compromise could be to name it `route` for the `Active` component only. "check if that route is active" sounds reasonable. A "target" would never be "active" anyway I guess.

**Comment by kate-shine** [comment]
True, that sounds weird. Oki, I'll rename it there as well

File: src/components/active.rs [hunk 1]
```diff
@@ -52,8 +52,8 @@ where
 
     let mut class = props.class.clone();
 
-    let active = match &props.target {
-        Some(target) => router.is_same(target),
+    let active = match &props.route {
+        Some(route) => router.is_same(route),
```
Comment (kate-shine) [inline src/components/active.rs:56]: Yep, in places like here, it really shows that the name fits better :)

**Review by ctron — APPROVED** [review]

**Pull request merged**